pkg/scale/varying_data_type.go

```diff
@@ -43,7 +44,7 @@ func mustNewVaryingDataTypeSliceAndSet(vdt VaryingDataType,
 	values ...VaryingDataTypeValue) (vdts VaryingDataTypeSlice) {
 	vdts = NewVaryingDataTypeSlice(vdt)
 	if err := vdts.Add(values...); err != nil {
-		panic(err)
+		panic(fmt.Errorf("%w: %s", ErrUnableToAddVaryingDataTypeValue, err))
```

Comment:
nit you can use Sprintf since we don't need to wrap the error in this case etc. since it's just a panic string

```suggestion
		panic(fmt.Sprintf("adding VaryingDataTypeValue: %s", err))
```